.github/workflows/semver-check-increment-npm.yml

```diff
@@ -1,6 +1,12 @@
 name: 'SemVer label Check and Increment package.json - npm'
 on:
-  workflow_call:
+  workflow_call: 
+    inputs:
+      mainLineBranchName: 
```

Comment:
We seem to have the standard of lowercase split by underscores, could this be changed to fit this?

Reply:
Had a look elsewhere in the repo and I think that the inputs use CAML case. Although given the yaml properties are generally snake case I can see why that looks a bit wrong!

Reply:
there are two standards in the repo it seems, one for workflows and one for actions
https://github.com/UKHomeOffice/sas-github-workflows/blob/main/.github/workflows/actions-check-dist.yml#L12
vs
https://github.com/UKHomeOffice/sas-github-workflows/blob/main/.github/actions/docker-publish/action.yml#L19